hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3APrefetchingInputStream.java

```diff
@@ -130,6 +131,7 @@ public void testReadLargeFileFully() throws Throwable {
     }
     // Verify that once stream is closed, all memory is freed
     verifyStatisticGaugeValue(ioStats, StreamStatisticNames.STREAM_READ_ACTIVE_MEMORY_IN_USE, 0);
+    assertDurationRange(ioStats, StoreStatisticNames.ACTION_EXECUTOR_ACQUIRED, 0, 10);
```

Comment:
This would pass for the duration of acquisition executor=0 as well. Think having an `assertThat.isGreaterThan(0)` should be fine as well.

Reply:
Done